### PR TITLE
Update the issue template

### PR DESCRIPTION
**Pull request by usha-mandya**
Signed-off-by: Usha Mandya <usha.mandya@docker.com>

Updated the issue template to help maintainers understand and address the issue quickly.

### CODE REVIEW

File: .github/ISSUE_TEMPLATE.md [hunk 1]
```diff
@@ -1,27 +1,36 @@
+
+## READ ME FIRST
+
+This repository is for reporting issues related to Docker Documentation. Before submitting a new issue, check whether the issue has already been reported. You can join the discussion using an emoji, or by adding a comment to an existing issue.
+
+You can ask general questions and get community support through the Docker Community Slack - http://dockr.ly/slack. Personalized support is available through the Docker Pro, Team, and Business subscriptions. See https://www.docker.com/pricing for details.
+
```
Comment (thaJeztah) [inline .github/ISSUE_TEMPLATE.md:7]: Do we want this part to be visible in the issue that's opened, or only visible to the person opening the ticket? If the latter, we could put this section as a HTML comment;

```
<!--
## READ ME FIRST

.....

-->
```

Comment (usha-mandya) [reply .github/ISSUE_TEMPLATE.md:7]: I think it helps if it's visible to the person opening the ticket. Let me update

Comment (usha-mandya) [reply .github/ISSUE_TEMPLATE.md:9]: Updated

File: .github/ISSUE_TEMPLATE.md [hunk 2]
```diff
@@ -1,27 +1,40 @@
...
+-->
+
 ### Problem description
 
-<!--Briefly describe the problem that you found.
-    Only DOCUMENTATION issues should be filed here.
-    For general questions, go to https://forums.docker.com/. -->
+<!--Briefly describe the problem that you found. A clear title and description helps us understand and address the issue quickly.
```
Comment (thaJeztah) [inline .github/ISSUE_TEMPLATE.md:14]: Perhaps slightly easier to read if we start the sentence on a new line

```suggestion
<!--
Briefly describe the problem that you found. A clear title and description helps us understand and address the issue quickly.
```

File: .github/ISSUE_TEMPLATE.md [hunk 3]
```diff
@@ -1,27 +1,40 @@
...
+
+Report only documentation issues here. Report any product issues in the corresponding product repository.
+
+If you are reporting a broken link issue, let us know how you arrived at the broken link; was it through a page on docs.docker.com, or from an external website?
+    -->
```
Comment (thaJeztah) [inline .github/ISSUE_TEMPLATE.md:19]: Probably works either way, but noticed this was indented

```suggestion
-->
```

**Review by thaJeztah — APPROVED** [review]
two small suggestions, but LGTM

**Review by thaJeztah — APPROVED** [review]
LGTM

**Pull request merged**